docs/futarchy/drawbacks.md

```diff
@@ -4,9 +4,9 @@ But of course, futarchy isn't perfect. These are some potential pitfalls with th
 
 ### Keynesian Beauty Contests
 
-Sometimes, investors buy what they think others will buy, not what they think the fundamentals support. The [GameStop short squeeze](https://en.wikipedia.org/wiki/GameStop\_short\_squeeze) is an example of this, and many more can be found in crypto.
```

Comment:
Changed as it was a rational short squeeze until the stock dilution and trading halts

Reply:
what I'm saying is that they weren't buying GME because they thought its discounted future cash flows were above its price

Reply:
I agree. But the extra nuances of the short squeeze distract the degen reader. I am more than happy to revert my changes should you wish.

Reply:
Sure